src/constants.js

```diff
@@ -121,6 +121,18 @@ export const itemStatusesTranslations = {
   [itemStatuses.RESTRICTED]: 'ui-requests.item.status.restricted',
 };
 
+export const requestTypeErrors = {
```

Comment:
Could you please rename from requestTypeErrors to REQUEST_TYPE_ERRORS?

Reply:
done